custom_components/tuya_local/devices/Fresco_Hydrate_Ultra_Pet_Fountain.yaml

```diff
@@ -0,0 +1,111 @@
+name: Hydrate Ultra
+products:
+  - id: cuw2k3ho3e25fade
+    name: Hydrate Ultra
+
+primary_entity:
+  entity: light
+  icon: "mdi:water-pump"
+  dps:
+    - id: 101
+      type: boolean
+      name: switch
+    - id: 102
+      name: brightness
+      type: integer
+      range:
+        min: 1
+        max: 10
+
+secondary_entities:
+  - entity: button
+    name: Reset Filter Alarm
+    icon: "mdi:air-filter"
+    category: config
+    dps:
+      - id: 104
+        type: integer
+        name: button
+  - entity: sensor
+    name: Filter life
+    icon: "mdi:air-filter"
+    category: diagnostic
+    dps:
+      - id: 104
+        type: integer
+        name: sensor
+        precision: 0
+        range:
+          min: 0
+          max: 43200
+        mapping:
+          - scale: 1440
+            invert: true
+        unit: d
+  - entity: button
+    name: Reset Pump Alarm
+    icon: "mdi:pump"
+    category: config
+    dps:
+      - id: 105
+        type: integer
+        name: button
+  - entity: sensor
+    name: Pump maintenance
+    icon: "mdi:pump"
+    category: diagnostic
+    dps:
+      - id: 105
+        type: integer
+        name: sensor
+        precision: 0
+        range:
+          min: 0
+          max: 86400
+        mapping:
+          - scale: 1440
+            invert: true
+        unit: d
+  - entity: sensor
+    name: Water quality
+    icon: "mdi:water"
+    category: diagnostic
+    dps:
+      - id: 106
+        type: integer
+        name: sensor
+        unit: TDS
+  - entity: switch
+    name: UV sterilization
+    icon: "mdi:flash"
+    category: config
+    dps:
+      - id: 107
+        type: boolean
+        name: switch
+  - entity: sensor
+    name: Water level
+    icon: "mdi:water"
+    category: diagnostic
+    dps:
+      - id: 108
+        type: integer
```

Comment:
dps_val appear to be strings, not integers.

Reply:
Its not a string but integer.
Uv runtime is no string.